common/logger.js

```diff
@@ -28,6 +28,7 @@ const transports = [
   new winston.transports.Syslog({
     level: config.sys_log_level || 'info',
     protocol: 'tcp4',
+    port: '1514',
```

Comment:
Please add the app_name property as well.

Reply:
Hi , I used the `config.broker_name` for the `app_name` as suggested in the email.
Best Regards

Reply:
Thanks @Espe0n 

Can you please take care of the boshrelease comments as well?